flac.yaml

```diff
@@ -0,0 +1,73 @@
+# Generated from https://git.alpinelinux.org/aports/plain/main/flac/APKBUILD
+package:
+  name: flac
+  version: 1.4.3
+  epoch: 0
+  description: Free Lossless Audio Codec
+  copyright:
+    - license: BSD-3-Clause AND GPL-2.0-or-later
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - automake
+      - autoconf
+      - cmake
+      - libogg-dev
+      - glibc-iconv
+      - samurai
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: 6c58e69cd22348f441b861092b825e591d0b822e106de6eb0ee4d05d27205b70
+      uri: https://downloads.xiph.org/releases/flac/flac-${{package.version}}.tar.xz
+
+  - runs: |
```

Comment:
Why not use `cmake/*` pipelines here?  Then you could drop the `samurai` and `cmake` deps.